vercel/vercel
# Update workflow for automation

In this PR, we are applying an overlay after pulling the openapi spec from https://openapi.vercel.sh/
This overlay fixes all the errrors that the speakeasy lint identifies when it runs, which stops it from being able to generate the SDK
- Pull source from URL
- Add lint to update `path-params` rule to warning only
- Add overlay to fix lint errors

How to test
- Clone the repo: https://github.com/vercel/vercel
- Change to this branch and make a copy `git checkout -b ....`
- `pnpm i` on the root should install speakeasy - otherwise https://www.speakeasy.com/docs/speakeasy-cli/getting-started
- CD into `packages/SDK` and do `speakeasy run`
- It should run with no errors (pass the lint) and then create a bunch of changes to update the SDK (this is why we make a new branch as we don't want to run speakeasy on this branch - once this merges, we will run it with the GH action)
